srcs/parse.c

```diff
@@ -22,6 +22,7 @@ char	*parse(t_line *inputs, char *line)
 {
 	char	**split;
 
+	// what is the use of this check you already verify beforehand whether or not the string is null
 	if (line[0] == 0)
```

Comment:
I check here:
```c
if (line[0] == '\0')
```
`line` can be malloc, have an address and have a string of one char, `'\0'`